panel/src/panel/upload.js

```diff
@@ -164,7 +176,7 @@ export default (panel) => {
 					// if upload should start immediately
 					this.set(options);
 					this.select(event.target.files);
-					this.start();
+					this.submit();
```

Comment:
```js
await this.submit();
```

It doesn't have to be like this, does it?

Reply:
I don't think so, because we want `this.input.remove()` to happen directly even while the upload is running.